sdk/ai/azure-ai-projects/azure/ai/projects/models/_enums.py

```diff
@@ -433,6 +433,13 @@ class VectorStoreChunkingStrategyResponseType(str, Enum, metaclass=CaseInsensiti
     STATIC = "static"
 
 
+class VectorStoreDataSourceAssetType(str, Enum, metaclass=CaseInsensitiveEnumMeta):
+    """Type of vector storage configuration."""
```

Comment:
We could add more description related to enterprise vector store configuration.

Reply:
Fixed.